files/magneticmediajs/info.ini

```diff
@@ -2,4 +2,4 @@ author = "Arnaud Leyder"
 github = "https://github.com/arnaudleyder/magneticmediajs"
 homepage = "https://www.magneticmediajs.com/"
 description = "JavaScript And CSS To Nicely Display Media Content"
-mainfile = "mjs.min.js"
\ No newline at end of file
+mainfile = "js/mjs.min.js"
```

Comment:
This files not going to exist with this pr (for `1.1.0` or any of `1.3`. It was correct before